### PR TITLE
Add unsafe flavor without ssl pinning

### PR DESCRIPTION
**Pull request by elgatovital**
Changes proposed in this pull request:
- Add unsafe flavor
- Built only for release candidates
- Internal-beta group will be notified
- No ssl pinning

@gnosis/mobile-devs

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -76,6 +77,27 @@ android {
...
+        unsafe {
+            applicationIdSuffix ".unsafe"
+            versionNameSuffix "-unsafe"
+            debuggable true
+            minifyEnabled false
+            signingConfig signingConfigs.debug
+            buildConfigField javaTypes.BOOLEAN, "VERBOSE_EXCEPTIONS", getKey("VERBOSE_EXCEPTIONS", "true")
+            buildConfigField javaTypes.BOOLEAN, "ALLOW_RESTRICTED_TX", getKey("ALLOW_RESTRICTED_TX", "true")
+            buildConfigField javaTypes.STRING, "NOTIFICATION_SERVICE_URL", asString(getKey("NOTIFICATION_SERVICE_URL", "https://safe-notification.staging.gnosisdev.com/api/"))
+            buildConfigField javaTypes.STRING, "SAFE_CREATION_FUNDER", asString(getKey("SAFE_CREATION_FUNDER", "0xAb8C18E66135561676f0781555D05CF6B22024A3"))
```
Comment (jpalvarezl) [inline app/build.gradle:89]: You could use `initWith debug` just modify what's different: https://developer.android.com/studio/build/build-variants

File: data/build.gradle [hunk 2]
```diff
@@ -23,6 +23,12 @@ android {
             buildConfigField javaTypes.STRING, "CLIENT_GATEWAY_URL", asString(getKey("CLIENT_GATEWAY_URL", "https://safe-client-rinkeby.staging.gnosisdev.com/"))
         }
 
+        unsafe {
+            buildConfigField javaTypes.STRING, "TRANSACTION_SERVICE_URL", asString(getKey("TRANSACTION_SERVICE_URL", "https://safe-transaction.staging.gnosisdev.com/api/"))
+            buildConfigField javaTypes.STRING, "RELAY_SERVICE_URL", asString(getKey("RELAY_SERVICE_URL", "https://safe-relay.staging.gnosisdev.com/api/"))
+            buildConfigField javaTypes.STRING, "CLIENT_GATEWAY_URL", asString(getKey("CLIENT_GATEWAY_URL", "https://safe-client-rinkeby.staging.gnosisdev.com/"))
```
Comment (jpalvarezl) [inline data/build.gradle:29]: It is using the old URL

**Review by biafra23 — APPROVED** [review]
LGTM 👍

File: data/build.gradle (hunk 2 above)
Comment (elgatovital) [reply data/build.gradle:29]: what's the new one for staging?

**Pull request merged**